source/reference/kubectl-minio-plugin.rst

```diff
@@ -0,0 +1,102 @@
+:orphan:
+
+.. _minio-kubectl-plugin:
+
+=======================
+MinIO Kubernetes Plugin
+=======================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+Overview
+--------
+
+.. admonition:: Current Stable Version is |operator-version-stable|
+   :class: note
+
+   This reference documentation reflects |operator-version-stable| of the 
+   MinIO Kubernetes Operator and :mc:`kubectl minio` plugin. 
+
+The :mc:`kubectl minio` plugin brings native support for deploying MinIO tenants
+to Kubernetes clusters using the ``kubectl`` CLI. You can use 
+:mc:`kubectl minio` to deploy a MinIO tenant with little to no interaction with 
+``YAML`` configuration files.
+
+.. image:: /images/Kubernetes-Minio.svg
```

Comment:
Here @prakashsvmx, made the text with the correct font. Also corrected some colors from the bird logo.
![Kubernetes-Minio-edit-01](https://user-images.githubusercontent.com/89046849/173852571-374e27ea-fe18-48da-8d5e-d88975e71177.svg)

Reply:
@prakashsvmx Had to redo the file because it was giving me some weird invisible blocks. Edited the text so it shows the addecuate font now and also fixed some colors.
![minio-k8s](https://user-images.githubusercontent.com/89046849/173855844-5ea8403b-4bd4-4041-88c5-e5ec199eb58a.svg)
[minio-k8s.svg.zip](https://github.com/minio/docs/files/8910419/minio-k8s.svg.zip)

Reply:
Swapped out the image. Thanks, @oscarocastellanos!